ui/package.json

```diff
@@ -93,7 +94,6 @@
     "@types/react-router": "5.1.18",
     "@types/react-router-dom": "5.3.3",
     "@types/react-table": "7.7.10",
-    "@types/sanitize-html": "2.6.2",
     "@typescript-eslint/eslint-plugin": "^5.41.0",
     "@typescript-eslint/parser": "5.11.0",
     "@viaa/avo2-components": "3.1.1",
```

Comment:
Duplicate dependency?

Reply:
You added that.
One is a peer dependency, but you added it to dev dependencies as well to make sure it is always installed in the developer machine.
although i ran into the issue a few times where i update the peer dependency and nothing changes, because i also need to change it in the dev dependency section